### ISSUE TITLE
Agent goes through surface

### ISSUE THREAD

**Issue by GameGC** [Unity-Technologies/NavMeshComponents#119]
![image](https://user-images.githubusercontent.com/32963287/54296664-6614ef80-45c6-11e9-9c2a-88039828b89f.png)
![image](https://user-images.githubusercontent.com/32963287/54296762-8775db80-45c6-11e9-8cce-a70892caa20d.png)

**Comment by turadr** [comment]
I see the blue surface is built for the _Humanoid_ agent. Is the NavMeshAgent which you have selected a _Humanoid_ or some other type?
If it's the same type as the purple surface then it will move on the flat purple surface.

**Comment by GameGC** [comment]
yes it is humanid,so 50% fix of this bug is disabling global navMesh,but you should do inside unity engine overiding one navMeshData with other and make it see normally each other
